streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/config/WebMvcConfig.java

```diff
@@ -70,6 +71,17 @@ public Module jacksonModule() {
 
   @Override
   public void addInterceptors(InterceptorRegistry registry) {
-    registry.addInterceptor(uploadFileTypeInterceptor).addPathPatterns("/flink/app/upload");
+    registry
+        .addInterceptor(uploadFileTypeInterceptor)
+        .addPathPatterns(INTERCEPTOR_UPLOAD_FILE_PATH_PATTERN);
```

Comment:
Maybe it should be inlined back to `"/flink/app/upload"` diractly.

Reply:
I have recovered